src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx

```diff
@@ -30,36 +31,37 @@ const SubNavigation: FC = () => {
             }
             isSuccess={isCopyTriggered}
           >
-            <Icon name="share-network" appearance={{ size: 'small' }} />
-            <span className="text-md ml-2">
+            <span className="text-md">
               {formatMessage({ id: 'members.subnav.invite' })}
             </span>
           </Tooltip>
-        </button>
+        </Button>
       </li>
       <li>
-        <button type="button" className={styles.navigationButton}>
-          <Icon name="lock-key" appearance={{ size: 'small' }} />
-          <span className="text-md ml-2">
+        <Button mode="senary" iconName="lock-key" iconSize="small">
+          <span className="text-md">
             {formatMessage({ id: 'members.subnav.permissions' })}
           </span>
-        </button>
+        </Button>
       </li>
       <li>
-        <button type="button" className={styles.navigationButton}>
-          <Icon name="pencil" appearance={{ size: 'small' }} />
-          <span className="text-md ml-2">
+        <Button
+          mode="senary"
+          onClick={onManageMembersClick}
+          iconName="pencil"
+          iconSize="small"
+        >
+          <span className="text-md">
             {formatMessage({ id: 'members.subnav.manage' })}
           </span>
-        </button>
+        </Button>
       </li>
       <li>
-        <button type="button" className={styles.navigationButton}>
-          <Icon name="address-book" appearance={{ size: 'small' }} />
-          <span className="text-md ml-2">
+        <Button mode="senary" iconName="address-book" iconSize="small">
+          <span className="text-md">
```

Comment:
I think, this style should be inside added to the button

Reply:
Moved to button styles